#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_automatic-replot-docstring.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## Introduce documentation string for the varaible automatic_replot

@DPATCH@

--- octave2.1-2.1.73.orig/doc/interpreter/plot.txi
+++ octave2.1-2.1.73/doc/interpreter/plot.txi
@@ -341,6 +341,8 @@
 any arguments.
 @end deffn
 
+@DOCSTRING(automatic_replot)
+
 Note that NaN values in the plot data are automatically omitted, and
 Inf values are converted to a very large value before calling gnuplot.
 
--- octave2.1-2.1.73.orig/doc/interpreter/var.txi
+++ octave2.1-2.1.73/doc/interpreter/var.txi
@@ -283,7 +283,7 @@
 Default value: @code{"+ "}.
 
 @item automatic_replot
-@xref{Two-Dimensional Plotting}.
+@xref{Low-Level plotting commands}.
 
 Default value: 0.
 
